src/tracer.ts

```diff
@@ -52,6 +55,60 @@ export class Tracer {
     // OpenTracing API methods
     // ---------------------------------------------------------------------- //
 
+    /**
+     * Gets the currently active span.
+     *
+     * @return the active Span. This is a shorthand for tracer.spanManager().active().
+     */
+    activeSpan(): Span | null {
+        return this.spanManager().active();
+    }
+
+    /**
+     * Starts a span, activates the span, executes the function, and finishes the span.
+     *
+     * @param {string} name the name of the operation.
+     * @param {SpanOptions} options options for the newly created span.
+     * @param {Function} f the function to execute during span activation.
+     * @return {A} the return value of the executed function.
+     * @template A
+     */
+    runSpan<A>(name: string, options: SpanOptions, f: () => A): A {
+        const span = this.startSpan(name, options);
+        try {
+            return this.spanManager().activate(span, f);
+        } finally {
+            span.finish();
+        }
+    }
+
+    /**
+     * Starts a span, activates the span, executes the async function, and finishes the span.
+     *
+     * @param {string} name the name of the operation.
+     * @param {SpanOptions} options options for the newly created span.
+     * @param {Function} f the function to execute during span activation.
+     * @return {A} the return value of the executed function.
+     * @template A
+     */
+    async runSpanAsync<A>(name: string, options: SpanOptions, f: () => Promise<A>): Promise<A> {
+        const span = this.startSpan(name, options);
+        try {
+            return await this.spanManager().activate(span, f);
+        } finally {
+            span.finish();
+        }
+    }
+
+    /**
+     * Gets the span manager.
+     *
+     * @return {SpanManager} - the span manager, which may be a noop.
+     */
+    spanManager(): SpanManager {
```

Comment:
I think it this could be called simply `scope()`. It makes sense to call `tracer.scope().span()` and `tracer.scope().run()` because these actions are indeed on the current scope.